requirements.txt

```diff
@@ -1 +1 @@
-git+https://github.com/astronomy-commons/hipscat.git@main
\ No newline at end of file
+git+https://github.com/astronomy-commons/hipscat.git@issue/84/add-sphgeom
```

Comment:
After review, and before merging, I will revert this change. Same applies for `docs/requirements.txt`.